src/TropicalGeometry/matrix.jl

```diff
@@ -74,3 +74,29 @@ function is_tropically_generic(A::MatrixElem{<:TropicalSemiringElem})
     return helper(transpose(A),nca,nra)
   end
 end
+
+@doc raw"""
+    is_polytrope(A::QQMatrix)
```

Comment:
```suggestion
    is_polytrope(A::QQMatrix, minOrMax::Union{typeof(min),typeof(max)})
```

Reply:
Additionally, the convention in the tropical part of OSCAR is that `min` is default.  If you think that is more confusing than helpful for polytrope, then please ignore that convention.  Otherwise please add it as follows `minOrMax::Union{typeof(min),typeof(max)}=min`.

Reply:
I think that is better, I have made the change